inference-engine/tests/unit/inference_engine/ngraph_graph_rewrite_test.cpp

```diff
@@ -0,0 +1,97 @@
+// Copyright (C) 2018-2020 Intel Corporation
```

Comment:
why do we have these tests inside IE, not inside ngraph?

Reply:
Moved to ngraph